waspc/data/Generator/templates/react-app/src/api.js

```diff
@@ -7,6 +7,10 @@ const api = axios.create({
 
 const WASP_APP_AUTH_TOKEN_NAME = "authToken"
 
+const storageKeys = [
```

Comment:
What is a bit tricky is that if we modify this code to have some more keys, a dev needs to remember to list them here, and they could forget to do so.

Small improvement would be to add a comment here that explains what this list is and why it is ipmortant to keep it up to date.

A bit better improvement could be to add some kind of abstraction that takes care of this, I am not sure what that could look like though. Maybe it allows setting only those keys that are defined here.

One option could be to also scope all wasp stuff under one big key, and then we always delete everything under that key. I am not sure if that is a bit weird / bad though, feels like it could use a bit of scrutiny. It is at the end also some kind of abstraction.

Reply:
I've updated the code to have an abstraction which introduces "key prefix" for each thing put in the `localStorage`.

We put stuff under `wasp:{key}` and remove only keys that start with `wasp:`